cmd/dm-syncer/main.go

```diff
@@ -36,13 +36,15 @@ func main() {
 	// 1. init conf
 	commonConfig := newCommonConfig()
 	conf, err := commonConfig.parse(os.Args[1:])
-	switch errors.Cause(err) {
-	case nil:
-	case flag.ErrHelp:
-		os.Exit(0)
-	default:
-		log.L().Error("parse cmd flags err " + err.Error())
-		os.Exit(2)
+	if err != nil {
+		switch errors.Cause(err) {
+		case nil:
```

Comment:
No need to keep the `case nil` here since `err != nil`... Why add the `if err != nil` anyway? `errors.Cause(nil) == nil`.

Reply:
you are right, update in 6800d8b